flink-staging/flink-streaming/flink-streaming-core/src/test/java/org/apache/flink/streaming/api/functions/PrintSinkFunctionTest.java

```diff
@@ -0,0 +1,336 @@
+
+		@Override
+		public Configuration getTaskConfiguration() {
+			return null;
+		}
+
+		@Override
+		public TaskManagerRuntimeInfo getTaskManagerInfo() {
+			return null;
+		}
+
+		@Override
+		public Configuration getJobConfiguration() {
+			return null;
+		}
+
+		@Override
+		public int getNumberOfSubtasks() {
+			return 0;
+		}
+
+		@Override
+		public int getIndexInSubtaskGroup() {
+			return 0;
+		}
+
+		@Override
+		public InputSplitProvider getInputSplitProvider() {
+			return null;
+		}
+
+		@Override
+		public IOManager getIOManager() {
+			return null;
+		}
+
+		@Override
+		public MemoryManager getMemoryManager() {
+			return null;
+		}
+
+		@Override
+		public String getTaskName() {
+			return null;
+		}
+
+		@Override
+		public String getTaskNameWithSubtasks() {
+			return null;
+		}
+
+		@Override
+		public ClassLoader getUserClassLoader() {
+			return null;
+		}
+
+		@Override
+		public Map<String, Future<Path>> getDistributedCacheEntries() {
+			return new Map<String, Future<Path>>() {
+				@Override
+				public int size() {
+					return 0;
+				}
+
+				@Override
+				public boolean isEmpty() {
+					return false;
+				}
+
+				@Override
+				public boolean containsKey(Object key) {
+					return false;
+				}
+
+				@Override
+				public boolean containsValue(Object value) {
+					return false;
+				}
+
+				@Override
+				public Future<Path> get(Object key) {
+					return null;
+				}
+
+				@Override
+				public Future<Path> put(String key, Future<Path> value) {
+					return null;
+				}
+
+				@Override
+				public Future<Path> remove(Object key) {
+					return null;
+				}
+
+				@Override
+				public void putAll(Map<? extends String, ? extends Future<Path>> m) {
+
+				}
+
+				@Override
+				public void clear() {
+
+				}
+
+				@Override
+				public Set<String> keySet() {
+					return null;
+				}
+
+				@Override
+				public Collection<Future<Path>> values() {
+					return null;
+				}
+
+				@Override
+				public Set<Entry<String, Future<Path>>> entrySet() {
+					return null;
+				}
+			};
+		}
+
+		@Override
+		public BroadcastVariableManager getBroadcastVariableManager() {
+			return null;
+		}
+
+		@Override
+		public AccumulatorRegistry getAccumulatorRegistry() {
+			return null;
+		}
+
+		@Override
+		public void acknowledgeCheckpoint(long checkpointId) {
+
+		}
+
+		@Override
+		public void acknowledgeCheckpoint(long checkpointId, StateHandle<?> state) {
+
+		}
+
+		@Override
+		public ResultPartitionWriter getWriter(int index) {
+			return null;
+		}
+
+		@Override
+		public ResultPartitionWriter[] getAllWriters() {
+			return new ResultPartitionWriter[0];
+		}
+
+		@Override
+		public InputGate getInputGate(int index) {
+			return null;
+		}
+
+		@Override
+		public InputGate[] getAllInputGates() {
+			return new InputGate[0];
+		}
+	};
+
+	public OutputStream out = new OutputStream() {
+		@Override
+		public void write(int b) throws IOException {
+
+		}
+	};
+
+	public PrintStream PrintStreamOriginal = null;
```

Comment:
This should be `private PrintStream PrintStreamOriginal = System.out;`. Then you don't need the null check later on and simply restore it in the `@After` method.